tests/integration/Owncloud/OcisPhpSdk/OcisTest.php

```diff
@@ -131,6 +131,48 @@ public function testAddUserToGroup(): void
         }
     }
 
+    /**
+     * @return void
+     */
+    public function testRemoveExistingUserFromGroup(): void
+    {
+        $token = $this->getAccessToken('admin', 'admin');
+        $ocis = new Ocis($this->ocisUrl, $token, ['verify' => false]);
+        $einsteinUserToken = $this->getAccessToken('einstein', 'relativity');
+        $einsteinUserOcis = new Ocis($this->ocisUrl, $einsteinUserToken, ["verify" => false]);
+        $users = $ocis->getUsers();
+        $philosophyHatersGroup =  $ocis->createGroup("philosophy-haters", "philosophy haters group");
+        $this->createdGroups = [$philosophyHatersGroup];
+        foreach($users as $user) {
+            $philosophyHatersGroup->addUser($user);
+        }
+        $initialMemberCount = count($philosophyHatersGroup->getMembers());
+        foreach ($users as $user) {
+            if($user->getDisplayName() === "Albert Einstein") {
+                $philosophyHatersGroup->removeUser($user);
+            }
+        }
+        $adminUserName = $users[0]->getDisplayName();
+        foreach ($ocis->getGroups(expandMembers: true) as $group) {
```

Comment:
there should be only one group, and if there were multiple groups the whole test would fail, so why having the `foreach`?

Reply:
I think the test coverage can be extended only if the issue regarding `getGroup` from another user is done.